rotkehlchen/assets/exchanges_mappings/__init__.py

```diff
@@ -0,0 +1,35 @@
+from rotkehlchen.assets.exchanges_mappings.binance import WORLD_TO_BINANCE  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.bitfinex import WORLD_TO_BITFINEX  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.bitpanda import WORLD_TO_BITPANDA  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.bitstamp import WORLD_TO_BITSTAMP  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.bittrex import WORLD_TO_BITTREX
+from rotkehlchen.assets.exchanges_mappings.coinbase import WORLD_TO_COINBASE  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.coinbase_pro import WORLD_TO_COINBASE_PRO  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.cryptocom import WORLD_TO_CRYPTOCOM  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.ftx import WORLD_TO_FTX  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.gemeni import WORLD_TO_GEMINI  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.iconomi import WORLD_TO_ICONOMI  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.kraken import WORLD_TO_KRAKEN  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.kucoin import WORLD_TO_KUCOIN  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.nexo import WORLD_TO_NEXO  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.poloniex import WORLD_TO_POLONIEX  # noqa: F401
+from rotkehlchen.assets.exchanges_mappings.uphold import WORLD_TO_UPHOLD  # noqa: F401
+
+__all__ = [
```

Comment:
I get the point of such files but the problem is you can no longer do

```python
 from rotkehlchen.assets.exchange_mappings.poloniex import WORLD_TO_POLONIEX
```

Due to this __init__.py the moment you do all are imported.

So do you really need this file?

Reply:
I did this for convenience but can change it. WIll do

Reply:
It's not that it';s required. It's just that this convenience ends up costing functionality. It's something I really dislike. Wish that python was smart enough to understand that if you did 

```
from rotkehlchen.assets.exchanges_mappings import X, Y, Z
```

you want the convenience

If not you don't.